src/utils/forecast.js

```diff
@@ -12,10 +12,29 @@ const forecast = (lat, lon, callback) => {
                 description: body.current.weather_descriptions[0],
```

Comment:
:point_up: use null. undefined should not be defined

Reply:
Could you please clarify what you mean?
Maybe the issue is outdated?